source/web_integration/pnapi_web_integration.rst

```diff
@@ -95,11 +128,11 @@ Client Side Parameters
       Set of JavaScript parameters to be captured where the key name should be RingPool parameter name.
       These values are updated every subsequent visit.
 
-  * - savePoolParams
+  * - updatePoolParams
     - Bool
-    - Default: `false`
+    - Default: `true`
 
-      When true, the integration code will not update `poolParams` on subsequent page visits.
+      When `false`, the integration code will not update `poolParams` on subsequent page visits.
```

Comment:
I'll let Garvan answer/clarify the above, but speaking of versions.. the PNAPI js is on a different versioning schema (2.1 ish), can you add the version number that is currently live (and therefore the documents are accurate for) on this page, probably near the top or near this parameters table?